website/pages/docs/secrets/databases/cassandra.mdx

```diff
@@ -74,8 +74,8 @@ the proper permission, it can generate credentials.
     lease_id           database/creds/my-role/2f6a614c-4aa2-7b19-24b9-ad944a8d4de6
     lease_duration     1h
     lease_renewable    true
-    password           8cab931c-d62e-a73d-60d3-5ee85139cd66
-    username           v-vaultuser-e2978cd0-
+    password           7iO-FxlVCdj8szsgGgsL
+    username           v_vaultuser_my_role_6zjahr4mjn5c9prgucy1_1602541475
```

Comment:
Not related to this PR in particular, but did we change the separator to `_` recently?

Reply:
No, it was always using underscores. However many of the docs were copy-pasted and had inaccurate username structures. I updated all of them to accurately represent what usernames will actually be generated.